### ISSUE TITLE
Suggestion - More customization in config

### ISSUE THREAD

**Issue by RunnyPilot** [DS-GAMING-SE/Sonic#21]
Loving the mod so far, however, I wish there were more options to enjoy this mod better. For example:

Chaos Emerald Scatter on/off - Allows the Chaos Emeralds to scatter if enabled. (DEFAULT ON)
Super Sonic Lifetime - 50 - Set how long you can stay as Super Sonic in seconds. Can be set to 0 for indefinite time, which allows you to de-transform with V. (DEFAULT 50)
Boost Lifetime - 10 - Set how long you can boost before running out in seconds. (DEFAULT 10)

You get the idea. I don't know if this has been suggested already, but I didn't see anything related, so thought I should post.

**Comment by Smonke-hue** [comment]
Yeah this would make the mod a clean 10/10 for me if it wasn't already

**Comment by adina-the-nerd** [comment]
I'd also like an option where if you go super rings scatter about the map & have rings scatter about the math in the single player as well. I think that'd be cool.

**Comment by floofytail22** [comment]
Adding to this, I think it would be useful if you could configure his base stats or the stats of his skills in some way.
